src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,40 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const {
+    userId,
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <>
+      <div className="PostInfo">
+        <div className="PostInfo__header">
+          <h3 className="PostInfo__title">
+            {title}
+          </h3>
+
+          <p>
+            {' Posted by  '}
+
+            <UserInfo user={user} key={userId} />
```

Comment:
Not needed to use key here. Add this attr in map method, when you need to render list of some items
```suggestion
            <UserInfo user={user} />
```